disk/assets/configuration/spec.yaml

```diff
@@ -51,6 +54,31 @@ files:
               type: array
               items:
                 type: string
+          - name: exclude_special_file_systems
+            description: |
+              Instruct the check to not collect from some common special file systems.
+              Works on Linux only.
+
+              The special file systems excluded with this option are:
+                - binfmt_misc
+                - configfs
+                - debugfs
+                - devtmpfs
+                - overlay
+                - proc
+                - rootfs
+                - securityfs
+                - sysfs
+                - tmpfs
+                - tracefs
+
+              These are added to the `file_system_blacklist`.
```

Comment:
The usage of blackllist is discouraged in favour of "exclusion list" or similar

Reply:
I agree with this but I feel like this should be done on a different PR (there are other checks that use this terminology and I think it's best to change them all at once). As it currently stands the name is `file_system_blacklist` so I think it would be confusing to use a different name (the only other option I see is rewording to avoid using the option name). Wdyt?